embedded-bins/kubernetes/Dockerfile

```diff
@@ -34,7 +35,9 @@ RUN \
     export KUBE_STATIC_OVERRIDES=$commands; \
   fi; \
   mkdir /out; \
+  export SOURCE_DATE_EPOCH=$SOURCE_DATE_EPOCH; \
   export FORCE_HOST_GO=y; \
+  export KUBE_VERBOSE=9; \
```

Comment:
Is this a deliberate change or is something that you commited accidentally? I'm not against it but I find this line surprising.

Reply:
Actually, this prints out the arguments for the build, which is quite handy and is what I used to verify that the build picks up a custom SOURCE_DATE_EPOCH:

https://github.com/k0sproject/k0s/actions/runs/6378582878/job/17309435491?pr=3542#step:7:11718

> Building binaries with GCFLAGS=all=-trimpath=/go/src/github.com/kubernetes/kubernetes/_output/local/go/src/k8s.io/kubernetes  ASMFLAGS=all=-trimpath=/go/src/github.com/kubernetes/kubernetes/_output/local/go/src/k8s.io/kubernetes LDFLAGS=all=-X 'k8s.io/kubernetes/vendor/k8s.io/client-go/pkg/version.buildDate=2023-10-02T09:52:01Z' -X 'k8s.io/kubernetes/vendor/k8s.io/component-base/version.buildDate=2023-10-02T09:52:01Z' -X 'k8s.io/client-go/pkg/version.buildDate=2023-10-02T09:52:01Z' -X 'k8s.io/component-base/version.buildDate=2023-10-02T09:52:01Z' -X 'k8s.io/kubernetes/vendor/k8s.io/client-go/pkg/version.gitCommit=89a4ea3e1e4ddd7f7572286090359983e0387b2f' -X 'k8s.io/kubernetes/vendor/k8s.io/component-base/version.gitCommit=89a4ea3e1e4ddd7f7572286090359983e0387b2f' -X 'k8s.io/client-go/pkg/version.gitCommit=89a4ea3e1e4ddd7f7572286090359983e0387b2f' -X 'k8s.io/component-base/version.gitCommit=89a4ea3e1e4ddd7f7572286090359983e0387b2f' -X 'k8s.io/kubernetes/vendor/k8s.io/client-go/pkg/version.gitTreeState=clean' -X 'k8s.io/kubernetes/vendor/k8s.io/component-base/version.gitTreeState=clean' -X 'k8s.io/client-go/pkg/version.gitTreeState=clean' -X 'k8s.io/component-base/version.gitTreeState=clean' -X 'k8s.io/kubernetes/vendor/k8s.io/client-go/pkg/version.gitVersion=v1.28.2+k0s' -X 'k8s.io/kubernetes/vendor/k8s.io/component-base/version.gitVersion=v1.28.2+k0s' -X 'k8s.io/client-go/pkg/version.gitVersion=v1.28.2+k0s' -X 'k8s.io/component-base/version.gitVersion=v1.28.2+k0s' -X 'k8s.io/kubernetes/vendor/k8s.io/client-go/pkg/version.gitMajor=1' -X 'k8s.io/kubernetes/vendor/k8s.io/component-base/version.gitMajor=1' -X 'k8s.io/client-go/pkg/version.gitMajor=1' -X 'k8s.io/component-base/version.gitMajor=1' -X 'k8s.io/kubernetes/vendor/k8s.io/client-go/pkg/version.gitMinor=28' -X 'k8s.io/kubernetes/vendor/k8s.io/component-base/version.gitMinor=28' -X 'k8s.io/client-go/pkg/version.gitMinor=28' -X 'k8s.io/component-base/version.gitMinor=28' -extldflags=-static -s -w